built_value_generator/lib/src/serializer_source_class.dart

```diff
@@ -72,6 +73,18 @@ abstract class SerializerSourceClass
   @memoized
   String get name => element.name;
 
+  // TODO(davidmorgan): this is not the right way to check, but is good enough
+  // while developing. Fix before publishing.
+  @memoized
+  bool get isNonNullByDefault =>
+      element.source.contents.data.contains('// @dart=2.9');
+
+  @memoized
+  String get orNull => isNonNullByDefault ? '?' : '';
+
+  @memoized
+  String get notNull => isNonNullByDefault ? '!' : '';
```

Comment:
Is `notNull` used? (I don't see it...)

Reply:
Not yet, I likely copied it in from a later PR :)